pkg/api/handlers/compat/containers_stats.go

```diff
@@ -80,6 +80,10 @@ func StatsContainer(w http.ResponseWriter, r *http.Request) {
 			ThrottlingData: docker.ThrottlingData{},
 		}
 	}
+	onlineCPUs, err := libpod.GetOnlineCPUs(ctnr)
+	if err != nil {
+		utils.InternalServerError(w, err)
```

Comment:
this needs a return after the InternalServerError call otherwise we just continue which will result in incorrect REST API responses.